tests/nnfw_api/src/one_op_tests/Rank.cc

```diff
@@ -17,26 +17,19 @@
 #include "GenModelTest.h"
 
 // WORKAROUND Handle int32_t type input/output
-union float_int {
```

Comment:
This file shows how to generate `TestCaseData` from now on.

- Use `uniformTCD` function if all input/output types are identical (`OneOp_Rank_Int32`)
- Otherwise manually build it (`OneOp_Rank`)